### PR TITLE
message fix to remove "block"

### PR DESCRIPTION
**Pull request by danalvrz**

### CODE REVIEW

**Review by davisagli — COMMENTED** [review]
@danalvrz Are we supposed to remove it for English too?

File: locales/en/LC_MESSAGES/volto.po [hunk 1]
```diff
@@ -27,7 +27,7 @@ msgid "Separator"
 msgstr ""
 
 #: components/schema
-# defaultMessage: Separator Block
+# defaultMessage: Separator
```
Comment (davisagli) [inline locales/en/LC_MESSAGES/volto.po:30]: @danalvrz The default message is generated automatically from where the message is defined in code, but I don't see a change to code. Did you update this manually, or did you miss committing the change to the file where the message is defined?

Comment (danalvrz) [reply locales/en/LC_MESSAGES/volto.po:30]: I just fixed it, and yes, I got the impression that the change was also for English

Comment (davisagli) [reply locales/en/LC_MESSAGES/volto.po:30]: @danalvrz yep, looks good now

**Comment by sneridagh** [comment]
@danalvrz @davisagli https://github.com/kitconcept/volto-separator-block/releases/tag/3.0.2

**Pull request merged**